fineract-provider/src/main/java/org/apache/fineract/infrastructure/dataqueries/service/ReadWriteNonCoreDataServiceImpl.java

```diff
@@ -19,6 +19,8 @@
 package org.apache.fineract.infrastructure.dataqueries.service;
 
 import static java.util.Arrays.asList;
+import static java.util.Locale.ENGLISH;
+import static org.apache.commons.lang3.time.DateFormatUtils.ISO_8601_EXTENDED_DATE_FORMAT;
```

Comment:
Please rather use `DateTimeFormatter.ISO_LOCAL_DATE`.

Reply:
I would, but the input of the validateColumn method is a string... And I didn't want to refactor the validateColumn method to accept  DateTimeFormatter. You ask too much @adamsaghy ! :moneybag: But I see want I can do ;)

Reply:
Its okay, leave it as is